installer-static.tpl.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+# This script is used to install the required packages for the application
+
+# Function for detect OS and set the package manager
+function detect_package_manager() {
+  ARCH=$(uname -m)
+
+  if yum --version > /dev/null 2>&1 || dnf --version > /dev/null 2>&1; then
+      echo "RhEL like"
+      install_yum_dnf
+  elif apt-get --version > /dev/null 2>&1; then
+      echo "Debian like"
+      install_apt
+  elif zypper --version > /dev/null 2>&1; then
+      echo "SuSe like"
+      install_zypper
+  else
+      echo "Unsupported OS"
+      exit 1
+  fi
+}
+
+install_apt ()
```

Comment:
I think a more suitable name is `setup_apt_repo`.

Reply:
fixed